tauri/Cargo.toml

```diff
@@ -63,6 +63,25 @@ create-dir = [ ]
 remove-dir = [ ]
 remove-file = [ ]
 rename-file = [ ]
+app-dir = [ ]
```

Comment:
I don't think we need a feature flag for each method; probably just one for all of them since it's the same dependency related.

Reply:
Yeah that makes more sense.